cypress.config.js

```diff
@@ -3,6 +3,8 @@ const { defineConfig } = require('cypress');
 module.exports = defineConfig({
   e2e: {
     setupNodeEvents(on, config) {
-    }
+    },
+    baseUrl: 'https://demoqa.com',
+    defaultCommandTimeout: 100000
```

Comment:
The `defaultCommandTimeout` is set to 100000 milliseconds (100 seconds), which is quite high. This might lead to unnecessarily long wait times during test execution. Consider setting it to a more reasonable value, such as 4000 or 5000 milliseconds, unless there is a specific reason for this high timeout.